gpflow/kernels.py

```diff
@@ -62,6 +62,37 @@ def __init__(self, input_dim, active_dims=None, name=None):
 
         self.num_gauss_hermite_points = 20
```

Comment:
I know that this is irrelevant to this PR but just saw that.
This is deprecated and should be removed. Kernels don't use `num_gauss_hermite_points` as far as I can tell.